aws-go-s3-folder/README.md

```diff
@@ -1,3 +1,5 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
```

Comment:
I have a silly question -- I expected the URL for this would need to look something like `https://app.pulumi.com/new?template=https%3A%2F%2Fgithub.com%2Fpulumi%2Fexamples%2Ftree%2Fmaster%2Faws-go-s3-folder`?

Reply:
> How else does the new project flow know which template to use?

HTTP referrer. The nice thing about it is it'll also do the right thing if viewing the README.md in a branch/commit -- it will fetch the template at that ref.